.gitattributes

```diff
@@ -0,0 +1,7 @@
+# Declare files that will always have CRLF line endings on checkout.
+*.sh text eol=lf
+*.py text eol=lf
+*.service text eol=lf
+Makefile text eol=lf
+configure text eol=lf
```

Comment:
(Optional) I don't believe we have `configure` or `.gitmodules` files in this repo.. is it necessary to list them anyway? Or do you use the same `.gitattributes` file across different projects and would prefer not to modify it?

Reply:
I just removed it, I copy pasted from another project.